README.md

```diff
@@ -77,18 +77,47 @@ To test against a local hydra instance
 - Initialising hydra iframe bridge creates a two way link between the hydra editor and your frontend. You will be able to optionally register call backs 
   for events allowing you to add more advanced editor functionality depending on your needs.
 
-# TODO: how to initialise the bridge.
-
-- Log into https://hydra.pretagov.com/ and paste in your local running frontend to test
+### How to initialise the bridge.
+
+- Import `initBridge` from [hydra.js](https://github.com/collective/volto-hydra/tree/hydra.js).
+- Call the `initBridge` and pass the origin of your adminUI as the argument to the initBridge method.
+- For example, if you are trying out demo editor, it will be: `https://hydra.pretagov.com`
```

Comment:
normally the frontend doesn't need to import hydra.js. The only time it does is when you log into the editor. You need to deal with this. Look at SB to see what it does. it loads the frontend using a special argument. 
This can be combined with the auth token so that the instructions are simpler? I don't think there is a need for the bridge to deal with the token at all. it shouldn't need to use the restapi
The shorter these instructions the better

Reply:
> Import of hydra.js is required for url tracking, and it is just one step of initiating the Bridge which i think is necessary.

It's only required after you login. the frontend wouldn't need to load it before you login. So SB adds a special param to the frontend url to signify it will start the edit mode or login with the editor. Then you can load the hydra.js.